app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -1787,14 +1787,18 @@ BrowserWorld::CreateSkyBox(const std::string& aBasePath, const std::string& aExt
     }
     return;
   }
+#if PICOXR
```

Comment:
This block of code mixes preprocessor directives with flow logic and is a bit hard to read.

Reply:
They're just two initializations, not sure how to make them more readable. Any suggestion?

Reply:
@svillar  I've tried to rewrite this code block but I don't think that this would be much of an improvement. Preprocessor directives are just kind of awkward 🫤 :

```cpp
#if PICOXR
  const std::string extension = aExtension.empty() ? ".png" : aExtension;
  GLenum glFormat = (extension == ".ktx") ? GL_COMPRESSED_RGB8_ETC2 : GL_SRGB8_ALPHA8;
#elif defined(OPENXR) && defined(OCULUSVR)
  const std::string extension = aExtension.empty() ? ".ktx" : aExtension;
  GLenum glFormat = GL_COMPRESSED_SRGB8_ETC2 : GL_RGBA8;
#else
  const std::string extension = aExtension.empty() ? ".ktx" : aExtension;
  glFormat = (extension == ".ktx") ? GL_COMPRESSED_RGB8_ETC2 : GL_SRGB8_ALPHA8;
#endif
```

Maybe just add some space between the different blocks so it is a bit easier to read?

```cpp
    ...
    return;
  }

#if PICOXR
  // Pico's OpenXR runtime does not support compressed textures at the moment. Use PNGs in the
  // meantime.
  const std::string extension = aExtension.empty() ? ".png" : aExtension;
  GLenum glFormat = GL_SRGB8_ALPHA8;
#else
  const std::string extension = aExtension.empty() ? ".ktx" : aExtension;
  GLenum glFormat = GL_RGBA8;
#endif

  if (extension == ".ktx") {
#if defined(OPENXR) && defined(OCULUSVR)
    glFormat =  GL_COMPRESSED_SRGB8_ETC2;
#else
    glFormat =  GL_COMPRESSED_RGB8_ETC2;
#endif
  }

  const int32_t size = 1024;
  if (m.skybox) {
    ...
```